Fixed some flow issues (#326)
Wyvest/SkyblockClient-REPO: thread.txt

[ATTACH type="full" alt="scthreaddivider.png"]2965072[/ATTACH]

[COLOR=rgb(184, 49, 47)][SIZE=5][U][B]One-click Forge Installation[/B][/U][/SIZE][/COLOR]

> [COLOR=rgb(0, 0, 0)][SIZE=4]This is self-explanitory

Reviewer: ```suggestion
[COLOR=rgb(0, 0, 0)][SIZE=4]Pretty much self-explanatory
```

Just choose your mods and packs, and click "Install SkyClient" to install.

> If you're using the Windows installer, you also won't need to install Java, as Minecraft comes with its own version by default.[/SIZE][/COLOR]

Reviewer: ```suggestion
You also probably won't need to install Java, as Minecraft comes with its own version by default.[/SIZE][/COLOR]
```

[COLOR=rgb(184, 49, 47)][SIZE=5][U][B]Update your mods and packs with very minimal effort[/B][/U][/SIZE][/COLOR]
We know how annoying it is to check your mods and packs for updates.

> We have a mod to fix this nusciance called "SkyClient Updater" that installs all the mods you have outdated as soon as you start up.

Reviewer: ```suggestion
We have a mod to fix this, SkyClient Updater. It automatically detects what mods you have, and if SkyClient has a newer version, it offers to download it for you.
```

[COLOR=rgb(184, 49, 47)][U][SIZE=5][B]Understand what your mods do and where they come from[/B][/SIZE][/U][/COLOR]
Many people simply do not know what the mods they have installed actually do, and download them just because a YouTuber recommended them.

> That's why we've added small guides for each mod we offer in our installer to understand if you want the features in the mod or not.
> Along with that, you can easily access the Github page, website, video, or whatever source the mod is from, by simply pressing the buttons next to it.

Reviewer: ```suggestion
That's why we've added links (like to the mod author's page or to the source code) and guides for every single thing in Skyclient, 
```

Reviewer 2: no it works out of the box

Reviewer: Java is installed by default on macos/linux. Besides the web installer exists.
